tests/test_vote_simple_dvt.py

```diff
@@ -0,0 +1,631 @@
+        ),
+    )
+
+    # VI. Update Oracle Report Sanity Checker parameters
+    validate_grant_role_event(evs[18], MAX_ACCOUNTING_EXTRA_DATA_LIST_ITEMS_COUNT_ROLE, agent, agent)
+    validate_grant_role_event(evs[19], MAX_NODE_OPERATORS_PER_EXTRA_DATA_ITEM_COUNT_ROLE, agent, agent)
+    validate_max_extra_data_list_items_count_event(evs[20], 4)
+    validate_max_operators_per_extra_data_item_count_event(evs[21], 50)
+
+    # scenario tests
+    validate_accounting_oracle_reports(simple_dvt, extra_data_service, stranger)
+    validate_manage_keys_role_members(stranger, protocol_permissions)
+
+
+def has_permission(permission: Permission, how: List[int]) -> bool:
+    return contracts.acl.hasPermission["address,address,bytes32,uint[]"](
+        permission.entity, permission.app, permission.role, how
+    )
+
+
+def validate_max_extra_data_list_items_count_event(event: EventDict, value: int):
+    _events_chain = [
+        "LogScriptCall",
+        "LogScriptCall",
+        "MaxAccountingExtraDataListItemsCountSet",
+        "ScriptResult",
+    ]
+
+    validate_events_chain([e.name for e in event], _events_chain)
+
+    assert event.count("MaxAccountingExtraDataListItemsCountSet") == 1
+    assert event["MaxAccountingExtraDataListItemsCountSet"]["maxAccountingExtraDataListItemsCount"] == value
+
+
+def validate_max_operators_per_extra_data_item_count_event(event: EventDict, value: int):
+    _events_chain = [
+        "LogScriptCall",
+        "LogScriptCall",
+        "MaxNodeOperatorsPerExtraDataItemCountSet",
+        "ScriptResult",
+    ]
+
+    validate_events_chain([e.name for e in event], _events_chain)
+
+    assert event.count("MaxNodeOperatorsPerExtraDataItemCountSet") == 1
+    assert event["MaxNodeOperatorsPerExtraDataItemCountSet"]["maxNodeOperatorsPerExtraDataItemCount"] == value
+
+
+def validate_simple_dvt_intialize_event(event: EventDict):
+    _events_chain = [
+        "LogScriptCall",
+        "ContractVersionSet",
+        "StuckPenaltyDelayChanged",
+        "Approval",
+        "LocatorContractSet",
+        "StakingModuleTypeSet",
+    ]
+
+    validate_events_chain([e.name for e in event], _events_chain)
+
+    assert event.count("ContractVersionSet") == 1
+    assert event.count("StuckPenaltyDelayChanged") == 1
+    assert event.count("Approval") == 1
+    assert event.count("LocatorContractSet") == 1
+    assert event.count("StakingModuleTypeSet") == 1
+
+    assert event["Approval"]["owner"] == contracts.simple_dvt.address
+    assert event["Approval"]["spender"] == contracts.burner.address
+    assert event["Approval"]["value"] == 2**256 - 1  # uint256 max
+
+    assert event["ContractVersionSet"]["version"] == 2
+    assert event["StuckPenaltyDelayChanged"]["stuckPenaltyDelay"] == SIMPLE_DVT_MODULE_STUCK_PENALTY_DELAY
+    assert event["LocatorContractSet"]["locatorAddress"] == contracts.lido_locator.address
+    assert event["StakingModuleTypeSet"]["moduleType"] == SIMPLE_DVT_MODULE_TYPE
```

Comment:
maybe extract these to `event_validators`

Reply:
looks like too specific check to be general